releases/v19.2.0-beta.20191014.md

```diff
@@ -0,0 +1,94 @@
+Get future release notes emailed to you:
+
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v19.2.0-beta.20191014.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v19.2.0-beta.20191014.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v19.2.0-beta.20191014.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach-unstable:v19.2.0-beta.20191014
+~~~
+
+### SQL language changes
+
+- The new `SHOW RANGE ... FOR ROW` statement identifies the range containing a specific row of a table or secondary index. This is particularly useful for validating the location of a specific piece of data when using [partitions](../v19.2/topology-geo-partitioned-replicas.html) to control data location for improved performance or GDPR compliance. [#40954][#40954]
+- The [`SHOW PARTITIONS`](../v19.2/show-partitions.html) statement now displays inherited as well as explicitly set zone configurations for partitions. Also, it is now possible to retrieve full zone configurations via the `full_config_sql` and `full_config_yaml` columns in the `crdb_internal.zones` table, and the new `zone_id` and `subzone_id` columns in the `crdb_internal.partitions` table point to partitions' corresponding zone configurations in `crdb_internal.zones`. [#40493][#40493]
+- Renamed the `config_yaml`, `config_sql`, and `config_proto` columns in the `crdb_internal.zones` table to `raw_config_yaml`, `raw_config_sql`, and `raw_config_proto`. [#40493][#40493]
+
+### Command-line changes
+
+- When `vmodule` is set to `exec_logs=2` to [log queries executed by a node](query-behavior-troubleshooting.html#per-node-execution-logs), CockroachDB-generated statements are no longer logged alongside client-generated statements. To log CockroachDB-generated statements as well, you must now use `exec_logs=3`. [#41149][#41149]
+- The [`cockroach demo`](../v19.2/cockroach-demo.html) command with `--nodes` set to `4` or higher runs an in-memory, multi-node cluster across multiple default regions. To simulate latencies between these default regions, you can now add the `--global` flag. Note that this flag is experimental and, as such, is hidden from the `cockroach demo --help` output and may change in future releases. [#40811][#40811]
```

Comment:
I would definitely not advertise in a release note a feature that is hidden because we're not yet confident it works properly!

Reply:
Also the behavior with `--nodes` is not different between 3 and 4 and beyond.
Even with 2 or 3 nodes you get nodes in different availability zones. The regions are immaterial.

What happens is that there's a list of 12 region+az combinations, and each value of `--nodes` will pick up one of them. It is a very primitive mechanism and we are likely to change this in the future, for example to enable 3-node demo clusters across multiple regions.